Formula/libopenmpt.rb

```diff
@@ -0,0 +1,60 @@
+require "English"
+
+class Libopenmpt < Formula
+  desc "Software library to decode tracked music files"
+  homepage "https://lib.openmpt.org/libopenmpt/"
+  url "https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-0.4.3+release.autotools.tar.gz"
+  version "0.4.3"
+  sha256 "d77443a279003921d6f0c4edb30d1e9dda387983f44113a6d58f623c1e6942ae"
+
+  depends_on "pkg-config" => :build
+
+  depends_on "portaudio"
+  depends_on "flac" => :recommended
```

Comment:
No options for formulas in Homebrew core. Either depend on it, or don't.

Reply:
Got it, thank you!